api/src/opentrons/protocol_api/module_contexts.py

```diff
@@ -79,6 +80,23 @@ def load_labware_object(self, labware: Labware) -> Labware:
         """
         mod_labware = self._geometry.add_labware(labware)
         self._ctx._implementation.get_deck().recalculate_high_z()
+
+        labware_namespace, labware_load_name, labware_version = labware.uri.split("/")
+        module_loc = self._geometry.parent
+        if isinstance(module_loc, (int, str)):
+            deck_slot = types.DeckSlotName.from_primitive(module_loc)
+        else:
+            raise Exception("Unexpected labware object parent")
```

Comment:
Same comment about this being more appropriate using `assert`, I think

```suggestion
        assert isinstance(module_loc, (int, str)), "Unexpected labware object parent"
        deck_slot = types.DeckSlotName.from_primitive(module_loc)
```

(Double checked the line length on this suggestion, won't trigger the linter)

Reply:
Ah yes. Will do. 
Thanks for checking the line length!